gems/spree/OSVDB-90865.yml

```diff
@@ -0,0 +1,20 @@
+---
+gem: spree
```

Comment:
This is also related to the spree_auth_devise gem.  Here is the [GitHub commit ] for the spree_auth_devise gem which is the same as the spree, another advisory might need to be added for that gem  ( https://github.com/spree/spree_auth_devise/commit/038d74771d3b5c13d13b738b73dfda1033a99f65 )with the fix